02 Requiements/UseCases/UseCase2/BriefUC2.md

```diff
@@ -0,0 +1,18 @@
+# Use case 2: Customize Order
+
+**Actors:**  
+Consultant
+
+**Description:**  
+The application prompts the user to specify additional components.  
+A Consultant specifies if the New Yorker wall has a door.  
+&nbsp;&nbsp;&nbsp; **-** A Consultant specifies which door type the New Yorker wall has.  
+&nbsp;&nbsp;&nbsp; **-** A Consultant specifies if the New Yorker wall has a Lockbox.  
+&nbsp;&nbsp;&nbsp; **-** A Consultant specifies if the New Yorker wall has a handle.  
+&nbsp;&nbsp;&nbsp; **-** A Consultant specifies which handle type the New Yorker wall has.  
+A Consultant specifies if the New Yorker wall has special glass.  
+&nbsp;&nbsp;&nbsp; **-** A Consultant specifies which special glass type the New Yorker wall has.  
+A Consultant specifies if the New Yorker wall is for a Wetroom.  
+A Consultant specifies if the New Yorker wall is a Shower wall.  
+The application redirects the Consultant to *Preview Order*.
```

Comment:
"Preview order" or a page where they can input contact info, because that is missing.